tasks/libs/common/utils.py

```diff
@@ -720,14 +734,13 @@ def is_pr_context(branch, pr_id, test_name):
 
 
 @contextmanager
-def collapsed_section(section_name):
+def section(section_name, collapsed=False):
     section_id = section_name.replace(" ", "_").replace("/", "_")
     in_ci = running_in_gitlab_ci()
     try:
         if in_ci:
-            print(
-                f"\033[0Ksection_start:{int(time.time())}:{section_id}[collapsed=true]\r\033[0K{section_name + '...'}"
-            )
+            collapsed = collapsed and '[collapsed=true]'
+            print(f"\033[0Ksection_start:{int(time.time())}:{section_id}{collapsed}\r\033[0K{section_name + '...'}")
```

Comment:
```suggestion
            is_collapsed = '[collapsed=true]' if collapsed else ''
            print(f"\033[0Ksection_start:{int(time.time())}:{section_id}{is_collapsed}\r\033[0K{section_name + '...'}")
```
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Might be wrong, but the way it's currently made would add a "False" string in the section wouldn't it?